ios/sdk/src/JitsiMeet.h

```diff
@@ -19,6 +19,9 @@
 
 #import <JitsiMeetSDK/JitsiMeetConferenceOptions.h>
 
+@class RCTReactNativeFactory;
```

Comment:
Move all the RN stuff to the implementation, this is the interface and we don't wanrt to expose that to embedders.